source/includes/_intents.md

```diff
@@ -41,22 +41,102 @@ Please do not share your access tokens with other users or applications and don'
 
```

Comment:
could you add a short into to what we'd use this endpoint for.
check here for inspiration: https://ultimateai.github.io/public-api/#chat-automation
basically something along the lined of "get the intent for a given body of text, extract entities etc."

Reply:
Not sure what you mean here. Is it not what I put at the top of the file? => 
`Send a visitor message from a chat or any type of text to get a list of matching intents with their confidences as well as a list of entities we extracted from the text. Use this endpoint to classify or label any piece of text.`

Reply:
you're right, didn't see that